packages/transaction-controller/src/helpers/PendingTransactionTracker.ts

```diff
@@ -59,7 +59,7 @@ export interface PendingTransactionTrackerEventEmitter extends EventEmitter {
 export class PendingTransactionTracker {
   hub: PendingTransactionTrackerEventEmitter;
 
-  #approveTransaction: (transactionId: string) => Promise<void>;
+  #approveTransaction: (transactionId: string) => Promise<ApprovalState>;
```

Comment:
Since we're not using the return value in this helper, could we keep the `Promise<void>` return to decouple them a little more and avoid the need to export the `ApprovalState` type?

We'd just have to update the construction in `TransactionController` to remove the return.

Reply:
Oh, you're right — good catch! Changed in [563ec54](https://github.com/MetaMask/core/pull/3827/commits/563ec545f619ce4a6c1d8158a7c7507d713628ca).